chain/emitter/slashing.go

```
@@ -0,0 +1,38 @@
+package emitter
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/slashing"
+)
+
+// handleEventSlash implements emitter handler for Slashing event.
+func (app *App) handleEventSlash(event EvMap) {
+	if raw, ok := event[slashing.AttributeKeyJailed]; ok && len(raw) == 1 {
+		consAddress, err := sdk.ConsAddressFromBech32(raw[0])
+		if err != nil {
```

Let's just ignore errors. We know it won't happen!